manifests/conf.pp

```diff
@@ -68,15 +69,13 @@
     }
   }
 
-  if $su_user and !defined('$su_group') {
-    $_su_user  = $su_user
-    $_su_group = 'root'
-  } elsif !defined('$su_user') and $su_group {
-    $_su_user  = 'root'
-    $_su_group = $su_group
-  } else {
-    $_su_user  = $su_user
-    $_su_group = $su_group
+  $_su_user = $su_user ? {
```

Comment:
should we drop this block and just set the default to 'root'? Same for the group.

Reply:
That would be much better.